packages/fuselage-ui-kit/src/blocks/VideoConferenceBlock/VideoConferenceBlock.tsx

```diff
@@ -105,6 +126,21 @@ const VideoConferenceBlock = ({
   const { data } = result;
   const isUserCaller = data.createdBy._id === userId;
 
+  const messageFooterText = getMessageFooterText(data.users.length);
+
+  const joinedUsernames = [...data.users]
+    .splice(0, MAX_USERS)
+    .map(({ name, username }) => (showRealName ? name : username))
+    .join(', ');
+
+  const iconTitle =
+    data.users.length > MAX_USERS
+      ? t('__usernames__and__count__more_joined', {
+          usernames: joinedUsernames,
+          count: data.users.length - MAX_USERS,
+        })
+      : t('__usernames__joined', { usernames: joinedUsernames });
```

Comment:
Shouldn't this respect the `useRealName` setting aswell?

Reply:
> Shouldn't this respect the `useRealName` setting aswell?

yep, let's replace it with the hook we're isolating in `ui-client`